src/platform/lumin-runtime/elements/builders/model-builder.js

```diff
@@ -168,6 +175,67 @@ export class ModelBuilder extends RenderBuilder {
         }
     }
 
+    _createSpinner (prism) {
```

Comment:
This code is duplicated in model and image.  Can we move it to a single helper utility?

Reply:
Yes, but not with this PR. I will be refactoring the code when applying @creationix 's polyfills update.

Reply:
ok